### PR TITLE
Add integration for Lektrico

### PR DESCRIPTION
**Pull request by mtarjoianu**
## Proposed change

Added integration for Lektrico Charging Station

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/60413
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2963
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lektrico.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ha_codeowners:
+  - '@mtarjoianu'
+ha_quality_scale: platinum
+ha_domain: lektrico
+ha_zeroconf: false
+ha_platforms:
+  - sensor
+---
+
+This `lektrico` integrates your [Lektrico Charging Station](https://lektri.co/en/product/ev-charging-station-lektri-co-svik-7-4kw-ac/) into your Home Assistant and allows you to monitor it.
```
Comment (frenck) [inline source/_integrations/lektrico.markdown]:
```suggestion
The Lektrico Charging Station integration integrates your [Lektrico Charging Station](https://lektri.co/en/product/ev-charging-station-lektri-co-svik-7-4kw-ac/) into your Home Assistant and allows you to monitor it.
```

File: source/_integrations/lektrico.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Configuration
+
+The Lektrico Charging Station device will be added as a sensor in Home Assistant.
+
+Before adding the integration you must know the device's IP.
+For this you can use some network scanners.
```
Comment (frenck) [inline source/_integrations/lektrico.markdown]:
```suggestion

The Lektrico Charging Station device will be added as a sensor in Home Assistant.

## Prerequisite

Before adding the integration, you must know the device's IP.

{% include integrations/config_flow.md %}

```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Additionally, there is a merge conflict as well, could you take a look?

**Comment by mtarjoianu** [comment]
> Additionally, there is a merge conflict as well, could you take a look?

I solved the confict, thank you.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**